tests/test___init__.py

```diff
@@ -274,10 +274,14 @@ def test__build_models(self, mocker):
             "__init__.py": models_init,
             f"{schema_1.reference.module_name}.py": schema_1_module_path,
             f"{schema_2.reference.module_name}.py": schema_2_module_path,
-            f"{enum_1.name}.py": enum_1_module_path,
-            f"{enum_2.name}.py": enum_2_module_path,
+            f"{enum_1.reference.module_name}.py": enum_1_module_path,
+            f"{enum_2.reference.module_name}.py": enum_2_module_path,
         }
-        models_dir.__truediv__.side_effect = lambda x: module_paths[x]
+
+        def models_dir_get(x):
+            return module_paths[x]
+
+        models_dir.__truediv__.side_effect = models_dir_get
```

Comment:
I assume this change was because it was difficult to read.  Maybe I should be using [itemgetter](https://docs.python.org/3.8/library/operator.html#operator.itemgetter) for this, would that be better / more Pythonic you think?

Pretty sure I do this in several tests.